schemas/eq_submission_schema.json

```diff
@@ -40,9 +38,13 @@
       "description": "The unique type identifier of this JSON file. Can be \"uk.gov.ons.edc.eq:surveyresponse\" or \"uk.gov.ons.edc.eq:feedback\"",
       "default": "",
       "examples": [
-        "uk.gov.ons.edc.eq:surveyresponse"
+        "uk.gov.ons.edc.eq:surveyresponse",
+        "uk.gov.ons.edc.eq:feedback"
       ],
-      "pattern": "^(.*)$"
+      "enum": [
+        "uk.gov.ons.edc.eq:surveyresponse",
+        "uk.gov.ons.edc.eq:feedback"
+      ]
```

Comment:
Do we want to consider splitting the schema and using partial?. At the moment with the current structure, a feedback type of version 0.0.3 with the data definition 0.0.3 i.e answers/lists would pass schema validation as it doesn't know list/answer would be only for `eq:surveyresponse` etc.